jms/app/pom.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus.ts.openshift</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+
+    <artifactId>jms-app</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Quarkus OpenShift TS: JMS: Artemis Application</name>
```

Comment:
One thing I'd like to understand is: why separate the app from the test module? I thought it's because you want to add a separate test that would use the QPid AMQP JMS client, but here, you clearly call this an "Artemis Application".

Reply:
Originally I wanted to have a one dead simple app depending only on provided javax (jakarta) JMS API. And then a separate module for let's say two implementations?

Looking at how it went, I would probably revise that approach :smiley: 

For this particular PR, I would leave it separated and I will iterate / refactor or merge into 1 app later.

IMHO

Reply:
Could be this app used with qpid extension?

Qpid extension is next in the row once few additional tests are added - client_ack + tx case.